src/Seq.App.Opsgenie/OpsgenieApp.cs

```diff
@@ -129,40 +140,69 @@ public void Dispose()
         public async Task OnAsync(Event<LogEventData> evt)
         {
             if (evt == null) throw new ArgumentNullException(nameof(evt));
+            var suppressDiff = DateTime.UtcNow - _lastAlert;
+            if (suppressDiff.TotalSeconds < _suppressionTime.TotalSeconds) return;
 
             var message = _generateMessage.Render(evt);
             var description = _generateDescription.Render(evt);
             var priority = ComputePriority(evt);
-            var tags = ComputeTags(evt);
+            var tags = ComputeTags(evt, _includeTags, _includeTagProperty, _tags);
             var responders = ComputeResponders(evt);
+            var notification = Guid.NewGuid().ToString("n");
 
             var alert = new OpsgenieAlert(
                 message,
                 evt.Id,
                 description,
                 priority.ToString(),
                 responders,
+                new Dictionary<string, string>
```

Comment:
This info should generally come from the event directly; e.g. based on the event type, events carry the URL that the user can navigate to in order to see relevant info. There are three branches in:

https://github.com/datalust/seq-app-htmlemail/blob/dev/src/Seq.App.EmailPlus/Resources/DefaultBodyTemplate.html#L105

based on the event type.

Alerts carry different data from regular events; the template shows some of this.

The Slack app has some helper types to achieve something along the lines of what you'll need in order to add this feature here:

https://github.com/bytenik/Seq.App.Slack/tree/master/src/Seq.App.Slack/Messages

Could be nice to bring into this codebase (but rather than as a "formatter", perhaps as "payload builders")? Seems deep enough to need a separate PR :)

Reply:
Yeah, can't recall if this was before I was across the alerting changes or not. It's useful to have the data show up in Opsgenie so I'll take a look at the slack helpers to start with

Reply:
Not 100% sure how to proceed with this. It looks like only NameSpacedAlertTitle and event URL would be affected? I think the latter would also have to be amended in the HandlebarsTemplate. 

Overall the goal would be to include the raw seq properties in the OpsGenieAlert, which makes this a more informative alert when examining in OpsGenie. Tested when I had access to Opsgenie but a bit harder to visualise now.